.travis.yml

```diff
@@ -23,21 +23,16 @@ branches:
 # install packages
 before_install:
   - ./bin/.travis/prepare_system.sh
-  - ./bin/.travis/prepare_sahi.sh
-#  - ./bin/.travis/prepare_selenium2.sh
+  - ./bin/.travis/prepare_selenium2.sh
 
 # setup requirements for running tests
 before_script:
   # Prepare eZ Publish (composer, permissions, assets, cache warmup)
   - ./bin/.travis/prepare_ezpublish.sh
-  # since setup wizard need to actually work to the other test suites to pass
-  # moved it to before_script so that the random timeout on package fetching
-  # don't fail travis
-  - php bin/behat --profile setupWizard --suite $INSTALL
```

Comment:
This one and the change above should continue to use Setup Wizard for community imho, platform installer is not written/intended to work on 5.x install.